src/main/kotlin/step1/StringCalculator.kt

```diff
@@ -0,0 +1,27 @@
+package step1
+
+import java.lang.RuntimeException
+
+private const val CUSTOM_SEPARATOR_FIND_REGEX = "//(.)\n(.*)"
+
+class StringCalculator {
+    companion object {
+        private val DEFAULT_SEPARATOR = "[,:]".toRegex()
+        fun calculate(input: String?): Int {
```

Comment:
공백라인을 활용하여 문맥을 분리해보세요. 😄 
```suggestion
        private val DEFAULT_SEPARATOR = "[,:]".toRegex()
        
        fun calculate(input: String?): Int {
```